### ISSUE TITLE
Get RStudio running under Windows and GNU/Linux for students

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/Cer2016#4]
Sent to CIT:

```
I would like to see that RStudio is available and functional for students in both GNU/Linux and Windows.

R is one of the top programming languages used in biology (next to Python and C++). RStudio is the most used (and free) IDE for R development.

I found out, on a student account, the following:

In the Windows environment, RStudio is installed incompletely: a student cannot even build a package

In the Xubuntu environment, RStudio is absent and the students do not have the rights to install it.

Because R programming is such a highly-required for our (including my) students I hope this will be fixed. I am willing to volunteer to check how well this works, but I'd need to have a student account for this.

Thanks, Richel Bilderbeek, p230198
```

**Comment by richelbilderbeek** [comment]
The Linux guys were first to answer:

```
Hallo Richel,

ik krijg via-via het verzoek om RStudio. Dat staat al in
/opt/netapps/bin/ op de LWP's.
Versie: 0.98. Is dat genoeg, of heb je wat nieuwers nodig?

Mvgr.
Jurjen
```

**Comment by richelbilderbeek** [comment]
Sent to Jurjen:

```
Hoi Jurjen,

Helaas pindakaas: In R mislukt het installeren van devtools.

Hier de staart van het installeren van devtools:

[...]
------------------------- ANTICONF ERROR ---------------------------
Configuration failed because libcurl was not found. Try installing:
* deb: libcurl4-openssl-dev (Debian, Ubuntu, etc)
* rpm: libcurl-devel (Fedora, CentOS, RHEL)
* csw: libcurl_dev (Solaris)
If libcurl is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a libcurl.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
ERROR: configuration failed for package ‘curl’
* removing ‘/home/s2536021/R/x86_64-pc-linux-gnu-library/3.2/curl’
Warning in install.packages :
  installation of package ‘curl’ had non-zero exit status
ERROR: dependency ‘curl’ is not available for package ‘httr’
* removing ‘/home/s2536021/R/x86_64-pc-linux-gnu-library/3.2/httr’
Warning in install.packages :
  installation of package ‘httr’ had non-zero exit status
ERROR: dependency ‘httr’ is not available for package ‘devtools’
* removing ‘/home/s2536021/R/x86_64-pc-linux-gnu-library/3.2/devtools’
Warning in install.packages :
  installation of package ‘devtools’ had non-zero exit status

The downloaded source packages are in
‘/tmp/RtmpoB5AAh/downloaded_packages’


Dit log geeft zelf de oplossing: installeer libcurl4-openssl-dev:

s2536021@pc-157-70:~$ sudo apt-get install libcurl4-openssl-dev
[sudo] password for s2536021: 
s2536021 is not allowed to run sudo on pc-157-70.  This incident will be reported.


Nadeel: studenten kunnen dit niet!

Kun jij dit goedmaken?

Dank, Richel
```

**Comment by richelbilderbeek** [comment]
Sent to Jurjen:

```
Hoi Jurjen,

Sorry, ik was wat korter, omdat mijn studenten er waren.

RStudio werkt prima. Maar packages maken is nog niet mogelijk. Omdat dit binnen R een (van de) geprefereerde werkwijzes is, zou dit wellicht mogelijk moeten kunnen zijn.

Te testen of alle faciliteiten er zijn kan met de R functie:

devtools::has_devel()

Dit geeft nu nog het antwoord 'FALSE'.

De package dit mist is libcurl4-openssl-dev. Zou je die beschikbaar kunnen maken voor studenten?

Alvast bedankt, zowieso al bedankt voor je hulp tot nu toe, Richel
```

**Comment by richelbilderbeek** [comment]
TLDR: Everything should be fixed now

From Jurjen:

```
Dat package staat nu in de te installeren lijst. Als er geen andere
nodig zijn, komt e.e.a. nu dus goed.
't Kan even duren tot 't overal geïnstalleerd is: twintig minuten op een
gewone, lopende PC (een kwartier tot het installscript draait plus wat
spreiding), en zeker twee keer zo lang op een PC die het sinds lange
tijd weer aan gaat (en dus een heleboel updates verwerken moet).

Mvgr.
Jurjen
```

**Comment by richelbilderbeek** [comment]
Got this screenshot from Johannes van Dijk:

![rs](https://cloud.githubusercontent.com/assets/2098230/15213371/195eddfe-1847-11e6-8f91-055f2693d9d3.png)

**Comment by richelbilderbeek** [comment]
RTools was missing.

I submitted a request to install RTools, so my next students can develop packages.
